pwiz_tools/Shared/Common/SystemUtil/ProcessRunner.cs

```diff
@@ -86,85 +86,94 @@ public class ProcessRunner : IProcessRunner
+                }
+                proc.WaitForExit();
+                int exit = proc.ExitCode;
+                if (exit != 0)
+                {
+                    line = proc.StandardError.ReadLine();
+                    if (line != null)
+                        sbError.AppendLine(line);
+                    if (sbError.Length == 0)
+                        sbError.AppendLine(@"Error occurred running process.");
+                    string processPath = Path.GetDirectoryName(psi.FileName)?.Length == 0
+                        ? Path.Combine(Environment.CurrentDirectory, psi.FileName)
+                        : psi.FileName;
+                    // ReSharper disable LocalizableElement
+                    sbError.AppendFormat("\r\nCommand-line: {0} {1}\r\nWorking directory: {2}{3}", processPath,
+                        // ReSharper restore LocalizableElement
+                        string.Join(" ", proc.StartInfo.Arguments), psi.WorkingDirectory,
+                        stdin != null ? "\r\nStandard input:\r\n" + stdin : "");
+                    throw new IOException(sbError.ToString());
+                }
 
-                        status = status.ChangeMessage(line);
+                // Make to complete the status, if the process succeeded, but never
+                // printed 100% to the console
+                if (percentLast < 100)
+                {
+                    status = status.ChangePercentComplete(100);
+                    if (status.SegmentCount > 0)
+                        status = status.NextSegment();
+                    if (progress != null)
                         progress.UpdateProgress(status);
-                    }
                 }
             }
-            proc.WaitForExit();
-            int exit = proc.ExitCode;
-            if (exit != 0)
-            {
-                line = proc.StandardError.ReadLine();
-                if (line != null)
-                    sbError.AppendLine(line);
-                if (sbError.Length == 0)
-                    sbError.AppendLine(@"Error occurred running process.");
-                string processPath = Path.GetDirectoryName(psi.FileName)?.Length == 0
-                    ? Path.Combine(Environment.CurrentDirectory, psi.FileName)
-                    : psi.FileName;
-                // ReSharper disable LocalizableElement
-                sbError.AppendFormat("\r\nCommand-line: {0} {1}\r\nWorking directory: {2}{3}", processPath,
-                // ReSharper restore LocalizableElement
-                    string.Join(" ", proc.StartInfo.Arguments), psi.WorkingDirectory,
-                    stdin != null ? "\r\nStandard input:\r\n" + stdin : "");
-                throw new IOException(sbError.ToString());
-            }
-            // Make to complete the status, if the process succeeded, but never
-            // printed 100% to the console
-            if (percentLast < 100)
+            finally

```

Comment:
I'm not sure why this diff is so ugly but all I did was put the above code inside a try block and added this finally block.

Reply:
Maybe you changed line endings somehow, or could it be indenting? It looks better if you ignore whitespace changes

Reply:
Ah indeed. I thought I had ignore whitespace enabled but maybe it resets.